walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim();
```

Comment:
Also here's an example of where you could split this PR if you wanted to - you can split Greeter and GreeterTest into their own PR, as it's like a separate 'feature'. It's fine that the resulting PR would be very small.